overlord/overlord.go

```diff
@@ -283,6 +297,12 @@ func (o *Overlord) Loop() {
 
 // Stop stops the ensure loop and the managers under the StateEngine.
 func (o *Overlord) Stop() error {
+	if o.udevMon != nil {
+		err := o.udevMon.Stop()
+		if err != nil {
+			logger.Noticef("Failed to stop udev monitor: %s", err)
```

Comment:
I suppose there is a reason for this but can you please add a comment that explains why we log the error and carry on rather than returning it?